sdk/storage/azure-storage-file-share/tests/test_file_async.py

```diff
@@ -914,7 +914,9 @@ def read(self, count):
     @AsyncStorageTestCase.await_prepared_test
     async def test_copy_file_async_private_file_async(self, resource_group, location, storage_account, storage_account_key, rmt, rmt_key):
         self._setup(storage_account, storage_account_key, rmt.name, rmt_key)
-        await self._create_remote_share()
```

Comment:
it seems previously create_remote_share() was to create a share in a different account. Are we still creating a share in another account in this commit?

Reply:
These commits aren’t organized. We are creating a remote share in this test ( look at the file)